settings/pyls-ms.vim

```diff
@@ -0,0 +1,32 @@
+augroup vimlsp_settings_pyls_ms
+  au!
+  LspRegisterServer {
+      \ 'name': 'pyls-ms',
+      \ 'cmd': {server_info->lsp_settings#get('pyls-ms', 'cmd', [lsp_settings#exec_path('pyls-ms')])},
+      \ 'root_uri':{server_info->lsp_settings#get('pyls-ms', 'root_uri', lsp_settings#root_uri(g:lsp_settings_root_markers))},
+	  \ 'initialization_options': lsp_settings#get('pyls-ms', 'initialization_options', {
+      \   'analysisUpdates': v:true,
+      \   'asyncStartup': v:true,
+      \   'displayOptions': {},
+      \   'interpreter': {
+      \      'properties': {
+	  \        'InterpreterPath': split(system('where python'), "\n")[0],
+      \        'UseDefaultDatabase': v:true,
+      \        'Version': matchstr(system('python -V'), '\s\zs\S\+'),
```

Comment:
This will match the trailing newline so you end up with something like `'Version': '3.8.1\n'`.

```diff
-'Version': matchstr(system('python -V'), '\s\zs\S\+'),
+'Version': trim(matchstr(system('python -V'), '\s\zs\S\+')),
```

Reply:
ditto